wls-monitoring-service/src/main/java/de/muenchen/oss/wahllokalsystem/monitoringservice/client/waehleranzahl/WaehleranzahlClientMapper.java

```diff
@@ -0,0 +1,24 @@
+package de.muenchen.oss.wahllokalsystem.monitoringservice.client.waehleranzahl;
+
+import de.muenchen.oss.wahllokalsystem.monitoringservice.eai.aou.model.WahlbeteiligungsMeldungDTO;
+import de.muenchen.oss.wahllokalsystem.monitoringservice.service.waehleranzahl.WaehleranzahlModel;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper
+public interface WaehleranzahlClientMapper {
+
+    ZoneOffset DEFAULT_ZONE = ZoneOffset.UTC;
```

Comment:
SystemOffset verwenden

```suggestion
    ZoneId DEFAULT_ZONE_ID = ZoneId.systemDefault();
```